src/frontpage/components/Offline/OfflineCarouselPlaceholder.tsx

```diff
@@ -0,0 +1,33 @@
+import classnames from 'classnames';
+import { IRefObject } from 'common/hooks/useRefMap';
+import React from 'react';
+import { DefaultEventImage } from '../../../events/components/DefaultEventImage';
```

Comment:
https://github.com/dotkom/onlineweb-frontend/wiki/(Opinionated)-Structuring-and-ordering-of-imports